core/shared/src/main/scala/clue/TransactionalClientImpl.scala

```diff
@@ -31,7 +31,7 @@ class TransactionalClientImpl[F[_]: MonadError[*[_], Throwable]: TransactionalBa
           val cursor = json.hcursor
           cursor
             .get[List[Json]]("errors")
-            .map(errors => new GraphQLException(errors.toString))
+            .map(errors => new ResponseException(errors))
```

Comment:
This was the nut of the problem.  I could only get at the actual errors as a `String` like

```
List({
  "message" : "'min' out of range: must be 1.0 <= min <= 3.0",
  "path" : [
    "updateConstraintSet"
  ],
  "locations" : [
    {
      "line" : 3,
      "column" : 9
    }
  ]
})
```

Reply:
Is this structure a standard?

Reply:
Yes, I believe so: https://spec.graphql.org/June2018/#sec-Errors

Reply:
Nice! I must have missed this. `path` and `locations` seem to be optional, could we model them as such? (Arguably, if present, they should be non-empty, especially `locations`, but I don't think this is reason enough to add `refined` to the API) Also, would it be too much to ask to implement `extensions` too?

Reply:
I made the `Decoder` accept missing fields and the `Encoder` skip otherwise empty fields.  Added `extensions`.